### PR TITLE
improve Primitives3D

### PR DESCRIPTION
**Pull request by tizianrot**
fixed docstrings for create_cylinder
fixed input list for create_box and create_polyhedron

### CODE REVIEW

File: pyaedt/modeler/cad/Primitives3D.py [hunk 1]
```diff
@@ -153,6 +153,8 @@ def create_cylinder(self, cs_axis, position, radius, height, numSides=0, name=No
         if isinstance(radius, (int, float)) and radius < 0:
             raise ValueError("Radius must be greater than 0.")
 
+        assert len(position) == 3, "Position Argument must be a valid 3 Element List"
```
Comment (PipKat) [inline pyaedt/modeler/cad/Primitives3D.py:156]: The message string should be a complete sentence using sentence case and punctuation and follow style guidelines: Position argument must be a valid three-element list. If you can somehow set off (tag) "Position" that would be fine.

**Comment by Samuelopez-ansys** [comment]
@tizianrot @gmalinve Please could you review the suggested modifications and push them? First update the branch please

File: pyaedt/modeler/cad/Primitives3D.py [hunk 2]
```diff
@@ -218,6 +218,7 @@ def create_polyhedron(
         References
         ----------
 
+
```
Comment (gmalinve) [inline pyaedt/modeler/cad/Primitives3D.py:221]: @tizianrot no need to add a new line.

File: pyaedt/modeler/cad/Primitives3D.py [hunk 3]
```diff
@@ -229,9 +230,10 @@ def create_polyhedron(
         ...                                              num_sides=8, name="mybox", matname="copper")
 
         """
+
```
Comment (gmalinve) [inline pyaedt/modeler/cad/Primitives3D.py:233]: @tizianrot no need to add a new line.

File: pyaedt/modeler/cad/Primitives3D.py [hunk 4]
```diff
@@ -229,9 +230,10 @@ def create_polyhedron(
...
         test = cs_axis
         cs_axis = GeometryOperators.cs_axis_str(cs_axis)
-        assert len(center_position) == 3, "Position Argument must be a valid 3 Element List"
+        assert len(center_position) == 3, "``Position`` Argument must be a valid 3 Element List"
```
Comment (gmalinve) [inline pyaedt/modeler/cad/Primitives3D.py:236]: @tizianrot the variable is called center_position not Position.

File: pyaedt/modeler/cad/Primitives3D.py [hunk 5]
```diff
@@ -257,7 +262,7 @@ def create_cone(self, cs_axis, position, bottom_radius, top_radius, height, name
         cs_axis : str
             Axis of rotation of the starting point around the center point.
             The default is ``None``, in which case the Z axis is used.
-        center_position : list, optional
+        position : list, optional
```
Comment (gmalinve) [inline pyaedt/modeler/cad/Primitives3D.py:265]: @tizianrot we shouldn't change variable names so we should get back to center_position.

**Comment by gmalinve** [comment]
@tizianrot there are still some things to fix in the already commited changes.
I see that there's still something to fix in other methods such as create_cylinder, create_polyhedron (you are asserting only the center position but there's also the start position), create_cone, create_bondwire, create_rectangle etc.

**Comment by Samuelopez-ansys** [comment]
@tizianrot Please could you implement the suggestions and push again? Thanks

File: pyaedt/modeler/cad/Primitives3D.py [hunk 6]
```diff
@@ -300,6 +303,7 @@ def create_cone(self, cs_axis, position, bottom_radius, top_radius, height, name
             raise ValueError("Top radius must be greater than 0.")
         if isinstance(height, (int, float)) and height <= 0:
             raise ValueError("Height must be greater than 0.")
+        assert len(position) == 3, "``Position`` Argument must be a valid 3 Element List"
```
Comment (PipKat) [inline pyaedt/modeler/cad/Primitives3D.py:306]: Message strings should be written as complete sentences, including punctuation.

The ``position`` argument must be a valid three-element list.

The need for a leading article here is that the parameter name starts with lower case, which looks odd.

**Comment by gmalinve** [comment]
Hi @PipKat, thanks for reviewing!

I just pushed the latest commit in order to apply your advice to all messages in that class.
Could you please double check that everything is fine now?

Thanks!

Giulia

**Review by PipKat — COMMENTED** [review]
@gmalinve I leave marking a PR as approved to someone with technical expertise. I only review the docstrings and prose--which looks fine. You'll need a technical team member to approve the PR.

**Comment by gmalinve** [comment]
@PipKat yes, I meant if you could review the doc since I changed other docstrings.
Thanks!

**Review by Samuelopez-ansys — APPROVED** [review]
Great work

**Pull request merged**